docker/data-science/notebooks/solr.py

```diff
@@ -270,7 +246,13 @@ def add_tag_request_handler(self, collection, request_name, field):
             "add-requesthandler" : {
                 "name": request_name,
                 "class": "solr.TaggerRequestHandler",
-                "defaults": {"field": field}
+                "defaults": {
+                    "field": field,
+                    "json.nl": "map",
+                    "sort": "popularity desc",
+                    "matchText": "true",
+                    "fl": "id,canonical_form,type,semantic_function,popularity,country,admin_area,*_p"
```

Comment:
I left a comment somewhere else to determine whether we should also be returning "surface_form". Need to verify we don't need it anywhere (maybe you already did?).

Reply:
Added to defaults

Reply:
In a future PR